Assets/SolarSystem/Asteroids/AsteroidMat.mat

```diff
@@ -130,4 +130,4 @@ Material:
       second: {r: 0.43, g: 0.43, b: 0.43, a: 1}
     - first:
         name: _LightPosition
-      second: {r: -0.019818218, g: -0.09801179, b: 1.495989, a: 0}
```

Comment:
This change doesn't seem to have anything to do with your PR and is probably a side effect of Unity's editor. There are pending changes in the Phone branch that will help with Unity's editor and material modification. Please remove this file from your PR.